tests/queries/test_explain.py

```diff
@@ -32,19 +32,13 @@ def test_basic(self):
         for idx, queryset in enumerate(querysets):
             for format in all_formats:
                 with self.subTest(format=format, queryset=idx):
-                    with CaptureQueriesContext(connection) as captured_queries:
-                        if queryset.query.select_for_update:
-                            with transaction.atomic():
-                                result = queryset.explain(format=format)
-                        else:
-                            result = queryset.explain(format=format)
-                            self.assertEqual(len(captured_queries), 1)
+                    with self.assertNumQueries(1), CaptureQueriesContext(
+                        connection
+                    ) as captured_queries:
```

Comment:
`_AssertNumQueriesContext(CaptureQueriesContext)` so this can effectively be replaced by 
```suggestion
                    with self.assertNumQueries(1) as captured_queries:
```